doc/proxy-metrics.md

```diff
@@ -182,14 +170,16 @@ Each of these metrics has the following labels:
                 pod to the proxy, or from the proxy to the application,
                `outbound` if the connection was established either from the
                 application to the proxy, or from the proxy to outside the pod.
+* `peer`: `src` if the connection was accepted by the proxy from the source,
+          `dst` if the connection was opened by the proxy to the destination.
 
 Note that the labels described above under the heading "Prometheus Collector labels"
 are also added to transport-level metrics, when applicable.
 
 ### Connection Close Labels
 
 The following labels are added only to metrics which are updated when a connection closes
-(`tcp_accept_close_total`, `tcp_connect_close_total`, `sent_bytes`, `received_bytes`, and 
+(`tcp_accept_close_total`, `tcp_connect_close_total`, `sent_bytes`, `received_bytes`, and
```

Comment:
metric names need updating. sent_bytes and received_bytes should be removed from this list